.github/workflows/ef-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: EF Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+permissions: read-all
+
+jobs:
+  ef-tests:
```

Comment:
I think that you could just put this job in the ci.yml file, after `tests`, you make it `needs: tests` and reuse the build from the previous job

Reply:
done.